.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove .idea

Reply:
Fixed in 119c22ffd55f371187c14fa3ebd11085eb7235f3